scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -141,6 +142,76 @@ def release_paths(self) -> List[Path]:
         return release_paths
 
 
+class ModelConfig(Config):
+    """
+
+    Example:
+
+    name: bert-base-uncased
+    path: # should contain local_path or should contain package object
+        local_path: "../models/bert-base-uncased" # the local path to the model
+        package:
+            url: https://huggingface.co/bert-base-uncased
+            commit_hash: 5546055f03398095e385d7dc625e636cc8910bf2
+    publish:
+        type: mlflow_model # could be one of (custom_model, mlflow_model, triton_model)
+        flavor: hftransformers # flavor should be specificed only for mlflow_model
+        tags: # tags published to the registry
+            isv : huggingface
+            task: fill-mask

```

Comment:
I assume this aligns with the spec we discussed? 

@novaturient95

Reply:
This is the asset.yaml file